index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title id="title">Elvis Presley</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body id="main">
+    <div id="img-div">
+      <img id="image" src="https://pymstatic.com/84696/conversions/frases-elvis-presley-wide.jpg" alt="">
+      <p id="img-caption" > “Values are like fingerprints. Nobody's are the same, but you leave 'em all over everything you do” </p>
+    </div>
```

Comment:
Instead of using the `<div>` tag for new sections of your info you should consider using the `<section>` tag instead since it is better for [SEO](https://frontendmasters.com/books/front-end-handbook/2018/learning/seo.html)